draft-ietf-ace-cwt-proof-of-possession.xml

```diff
@@ -237,22 +233,14 @@
 	  representations.
 	</t>
 	<t>
-	  The <spanx style="verb">cnf</spanx> claim value MUST represent only a single
-	  proof-of-possession key.  At most one of the
-	  <spanx style="verb">COSE_Key</spanx>
-	  and <spanx style="verb">Encrypted_COSE_Key</spanx>
-	  <!-- and <spanx style="verb">jku</spanx> (JWK Set URL) -->
-	  confirmation values defined in <xref target="fig:cborMappings"/> may be present.
-	  Note that if an application needs to represent multiple proof-of-possession
-	  keys in the same CWT, one way for it to achieve this is to
-	  use other claim names, in addition to <spanx style="verb">cnf</spanx>,
-	  to hold the additional proof-of-possession key information.
-	  These claims could use the same syntax and semantics
-	  as the <spanx style="verb">cnf</spanx> claim.
-	  Those claims would be defined by applications or other specifications
-	  and could be registered in
-	  the IANA "CBOR Web Token Claims" registry <xref target="IANA.CWT.Claims"/>.
-	</t>
+	The <spanx style="verb">cnf</spanx> claim value may contain one or multiple proof-of-
+    possession keys. Hence, more than one <spanx style="verb">jwk</spanx>, <spanx style="verb">jwe</spanx>, and/or <spanx style="verb">jku</spanx> (JWK
+    Set URL) confirmation values  may be present in a CWT.  A profile may
+ impose
+    further restrictions on the number of PoP keys that are present and
+    also whether all or a only a subset of them needs to be utilized by in
+    a given application domain.
+</t>
```

Comment:
This change introduces unnecessary syntactic and semantic ambiguity.  It also breaks the semantic equivalence with RFC 7800.  Please remove this change.

Reply:
+1
If this is really needed it would be good to first have consensus by the WG. If there are no strong use cases I would prefer to keep this limitation since it makes implementing so much easier.